src/p5.dance.js

```diff
@@ -1499,6 +1546,17 @@ module.exports = class DanceParty {
     console.warn(message);
   }
 
+  setExternalLayerSource(dancerName) {
+    if (!this.externalLayer) {
+      return;
+    }
+    const base = 'https://curriculum.code.org/media/musiclab/generate/dancers/';
```

Comment:
Longer term, I think it would make sense for the main repo to know about asset URLs, with `dance-party` only having a concept of a dancer ID/metadata.